### PR TITLE
fix(knownIssues) add troubleshoot and known issues

### PR DESCRIPTION
**Pull request by rbioteau**
* For certificates issues in the studio
* For rich text editor on Linux in the Studio

### CODE REVIEW

File: modules/ROOT/pages/bonita-bpm-studio-installation.adoc [hunk 1]
```diff
@@ -190,3 +190,8 @@ A given Bonita Studio version can only run with some specifics Java versions (xr
 If you encounter some issues related to the JVM during the installation, ensure first that a valid Java version is available. You can for exemple type _java -version_ on a terminal, and informations about the main Java version used will be displayed. +
 If the problem persist, the easiest way to solve it is to remove the installed Java versions, to download that last stable build of the required Java version https://www.oracle.com/technetwork/java/javase/downloads/index.html[on the Oracle website] and to install it properly. +
 Those issues can be related to bad Java installations, or to incompatibilities between a given Java version and Bitrock (Bitrock is the install builder used to install Bonita, it has the responsibility to detect the JRE used to install Bonita. Some defects on a given version could lead to incompatibility).
+
```
Comment (NathalieC) [inline modules/ROOT/pages/bonita-bpm-studio-installation.adoc:193]: On row 143 (Troubleshooting), we would like to consistently identify the Troubleshooting sections starting in 2021.2.
This is why we use == image:images/troubleshooting.png[troubleshooting-icon] Troubleshooting
If you need to use it before 2021.2, you can add the image to this version too.

File: modules/ROOT/pages/release-notes.adoc [hunk 2]
```diff
@@ -369,3 +369,11 @@ A standalone docker image that package the LDAP Synchronizer was released under
 * BPO-664 - Security risk: API/formsDocumentImage executes injected script
 * BR-454 - Sometimes CallActivity are left Completed without target process
 * RUNTIME-50/BR-621 - All LDAP group objectclass checks are now case-insensitive.
+
+== Known issues
```
Comment (NathalieC) [inline modules/ROOT/pages/release-notes.adoc:373]: Do we really need to make a difference between troubleshooting and known issue (that will create a trouble, that we will help to shoot)? 
If not, then I'd advise to use Troubleshooting everywhere, with the icon:
== image:images/troubleshooting.png[troubleshooting-icon] Troubleshooting
With the addition of the troubleshooting icon in the 2021.1 folder.

File: modules/ROOT/pages/bonita-bpm-studio-installation.adoc [hunk 3]
```diff
@@ -190,3 +190,8 @@ A given Bonita Studio version can only run with some specifics Java versions (xr
 If you encounter some issues related to the JVM during the installation, ensure first that a valid Java version is available. You can for exemple type _java -version_ on a terminal, and informations about the main Java version used will be displayed. +
 If the problem persist, the easiest way to solve it is to remove the installed Java versions, to download that last stable build of the required Java version https://www.oracle.com/technetwork/java/javase/downloads/index.html[on the Oracle website] and to install it properly. +
```
Comment (NathalieC) [inline modules/ROOT/pages/bonita-bpm-studio-installation.adoc]:
```suggestion
If the problem persists, the easiest way to solve it is to remove the installed Java versions, to download the last stable build of the required Java version https://www.oracle.com/technetwork/java/javase/downloads/index.html[on the Oracle website] and to install it properly. +
```

File: modules/ROOT/pages/release-notes.adoc (hunk 2 above)
Comment (rbioteau) [reply modules/ROOT/pages/release-notes.adoc:373]: For me, there is a semantic difference between both.

**Pull request merged**